libraries/launcher/org/prismlauncher/log4jinjector/UniversalLog4JInjector.java

```diff
@@ -0,0 +1,41 @@
+package org.prismlauncher.log4jinjector;
+
+import java.util.logging.Logger;
+import java.util.logging.Level;
+
+public class UniversalLog4JInjector {
+    private static final Logger LOGGER = Logger.getLogger("UniversalLog4JInjector");
+
+    public static void inject() {
+        if (System.getProperty("prism.log4j.inject").equals("false")) {
```

Comment:
This causes a NullPointerException. I also think "org.prismlauncher" would make more sense. While Boolean.getBoolean could be used, that would make it opt-in which would not be good.
```suggestion
        if ("false".equalsIgnoreCase(System.getProperty("org.prismlauncher.log4j.inject"))) {
```